### PR TITLE
LG-150 Drupal Upgrade

### PR DESCRIPTION
**Pull request by TaylorAkin**
- added patch for drush 12 and ckeditor compatability
 - removes deprecated modules
 - unused Paragraphs Inline Entity Form
 - replaces CKEditor Entity Link with linkit
 - Fixes time issue with views content

## Summary / Approach

Drupal 10 upgrade

## Testing Instruction(s)

Literally regression test everything

## Screenshots

Screenshot 2023-10-26 at 11 23 32 AM (2)

## Metadata

| Question | Answer |
|----------|--------|
| Did you perform a self-review of this PR? | Yes
| Documentation reflects changes? | n/a
| Unit/Functional tests reflect changes? | n/a
| Did you perform browser testing? | Yes
| Did you provide detail in the summary on how and where to test this branch? | Yes
| Relevant links | JIRA issue, bug reports, security alerts, etc. https://apaxsoftware.atlassian.net/jira/software/c/projects/LG/boards/68?selectedIssue=LG-150

### CODE REVIEW

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-504-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-504](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-504) for lexky-d8.

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-504-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-504](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-504) for lexky-d8.

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-504-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-504](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-504) for lexky-d8.

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-504-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-504](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-504) for lexky-d8.

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-504-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-504](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-504) for lexky-d8.

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-504-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-504](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-504) for lexky-d8.

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-504-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-504](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-504) for lexky-d8.

**Comment by TaylorAkin** [comment]
[Visit Site](https://pr-504-lexky-d8.pantheonsite.io/)

Created multidev environment [pr-504](https://dashboard.pantheon.io/sites/085efc9d-6f4f-4d02-a80b-9d5cf086a982#pr-504) for lexky-d8.

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -80,10 +81,10 @@ module:
   options: 0
   page_cache: 0
   pantheon_advanced_page_cache: 0
-  paragraphs_inline_entity_form: 0
   paragraphs_jquery_ui_accordion: 0
   path: 0
   path_alias: 0
+  phpass: 0
```
Comment (JMStewart00) [inline config/sync/core.extension.yml:87]: gif

File: config/sync/metatag.metatag_defaults.front.yml [hunk 2]
```diff
@@ -13,7 +13,6 @@ tags:
   icbm: '38.044810, -84.495460'
   canonical_url: '[site:url]'
   geo_placename: 'Lexington Kentucky'
-  news_keywords: 'City of Lexington KY'
```
Comment (JMStewart00) [inline config/sync/metatag.metatag_defaults.front.yml:16]: Just curious why this was removed.

File: config/sync/views.view.content.yml [hunk 3]
```diff
@@ -258,8 +258,8 @@ display:
           label: Updated
           exclude: false
           alter:
-            alter_text: true
-            text: '{% include ''@lex/includes/time.html.twig'' with { notag: true, time: (changed | date("m/d/Y h:i a") ) } %}'
+            alter_text: false
+            text: ''
```
Comment (JMStewart00) [inline config/sync/views.view.content.yml:262]: Do we need to check and compare the content view? /admin/content?

File: config/sync/metatag.metatag_defaults.front.yml (hunk 2 above)
Comment (TaylorAkin) [reply config/sync/metatag.metatag_defaults.front.yml:16]: Looks like google dropped it.

https://www.drupal.org/project/metatag/issues/2973351

Comment (JMStewart00) [reply config/sync/metatag.metatag_defaults.front.yml:16]: Word.

**Review by JMStewart00 — APPROVED** [review]

**Pull request merged**